### ISSUE TITLE
build fails on alpine linux ppc64le

### ISSUE THREAD

**Issue by ncopa** [voidlinux/xbps#291]
````
make[1]: Entering directory '/home/ncopa/aports/community/xbps/src/xbps-0.52/include'
sed -e "s|@@VERSION@@|0.52|g" xbps.h.in > xbps.h
make[1]: Leaving directory '/home/ncopa/aports/community/xbps/src/xbps-0.52/include'
make[1]: Entering directory '/home/ncopa/aports/community/xbps/src/xbps-0.52/lib'
 [CC]		external/fexec.o
In file included from /usr/include/signal.h:48:0,
                 from /usr/include/sys/wait.h:25,
                 from external/fexec.c:35:
/usr/include/bits/signal.h:27:23: error: expected identifier or '(' before '[' token
  unsigned long _unused[4];
                       ^
make[1]: *** [Makefile:72: external/fexec.o] Error 1
make[1]: Leaving directory '/home/ncopa/aports/community/xbps/src/xbps-0.52/lib'
make: *** [Makefile:15: all] Error 1
````

What happens here is that `_unused` is passed as a define on the command line from configure:
````sh
echo "CPPFLAGS +=   -D_unused=\"__attribute__((__unused__))\"" >>$CONFIG_MK
````

this messes up the compilation of  musl's `/usr/include/bits/signal.h`:
````c
typedef struct sigcontext {
        unsigned long _unused[4];
        int signal;
        int _pad0;
        ...
````

**Comment by ncopa** [comment]
From [glibc manual](https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html):
> In addition to the names documented in this manual, reserved names include all external identifiers (global functions and variables) that begin with an underscore (‘_’) and all identifiers regardless of use that begin with either two underscores or an underscore followed by a capital letter are reserved names. This is so that the library and header files can define functions, variables, and macros for internal purposes without risk of conflict with names in user programs.

From [POSIX](http://pubs.opengroup.org/onlinepubs/009695399/functions/xsh_chap02_02.html):
> `With the exception of identifiers beginning with the prefix _POSIX_, all identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use by the implementation.`

I read that as all identifiers that begin with an underscore is reserved for any use by the libc.

**Comment by chneukirchen** [comment]
Yes, this is a namespace violation by xbps. I propose to use `UNUSED` instead.
